libs/ui/src/bmd_paper_ballot.test.tsx

```diff
@@ -117,9 +117,6 @@ test('BmdPaperBallot renders votes for candidate contests and yes-no contests',
   screen.getByText(
     hasTextAcrossElements(/Question B: Separation of Powers.?No/)
   );
-
-  // Use a snapshot to avoid unintentional regressions to general layout
-  expect(container).toMatchSnapshot();
```

Comment:
Just making sure I understand, we can remove this because rendering and snapshot testing was moved to the backend?

Reply:
yeah, it's usefulness was a little questionable to begin with, since it was just comparing html and css code (we've been stripping out this type of snapshot from tests whenever possible) and it's redundant now that we have pixel-based diffing in the server-side tests now